datafusion/expr/src/expr_fn.rs

```diff
@@ -25,11 +25,17 @@ use crate::{
     ScalarFunctionImplementation, ScalarUDF, Signature, StateTypeFunction, Volatility,
 };
 use arrow::datatypes::DataType;
-use datafusion_common::Result;
+use datafusion_common::{Column, Result};
 use std::sync::Arc;
 
 /// Create a column expression based on a qualified or unqualified column name
-pub fn col(ident: &str) -> Expr {
+///
+/// example:
```

Comment:
This is demonstrating you can still call `col` with a `&str` but now also with a `String` or `&String` to allow backwards compatibility as well as reuse